webapp-django/crashstats/api/views.py

```diff
@@ -258,7 +259,7 @@ def model_wrapper(request, model_name):
     form = FormWrapper(model, request.REQUEST)
     if form.is_valid():
         try:
-            result = function(**form.cleaned_data)
+            result = function(**dict(form.cleaned_data, _user=request.user))
```

Comment:
Pretty sure you can just do:

```
result = function(_user=request.user, **form.cleaned_data)
```

You don't need to form a dict and then de-dict it.